src/Twig/FileLoader.php

```diff
@@ -96,12 +96,48 @@ public function findTemplate(string $name): string
             return $name;
         }
 
-        try {
-            $path = $this->twigView->resolveTemplatePath($name);
-        } catch (MissingTemplateException $e) {
-            throw new LoaderError($e->getMessage());
+        [$plugin, $name] = pluginSplit($name);
+        $name = str_replace('/', DIRECTORY_SEPARATOR, $name);
+
+        if ($plugin !== null) {
+            $path = $this->checkExtensions(Plugin::templatePath($plugin) . $name);
+            if ($path !== null) {
+                return $path;
+            }
+
+            throw new LoaderError("Could not find template `{$name}` in plugin `{$plugin}`.");
```

Comment:
Would be extra nice to include the paths that were searched.

Reply:
Sure. Done.